src/tutorials/0005/06.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <FileLesson
+    :text="text"
+    :code="code"
+    :validate="validate"
+    :overrideErrors="true"
+    :modules="modules"
+    :exercise="exercise"
+    :solution="solution" />
+</template>
+
+<script>
+import FileLesson from '../../components/FileLesson'
+import text from './06.md'
+import exercise from './06-exercise.md'
+
+const code = `/* global ipfs */
+const run = async (files) => {
+  const fileObjectsArray = files.map((file) => { return { path: file.name, content: file }})
```

Comment:
I think shorter notation should work? (here and in line 32)
```suggestion
  const fileObjectsArray = files.map((file) => { path: file.name, content: file })
```